getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Events"
+weight: 20
+---
+
+# Events
+
+
+### Introduction
+
+Events are portions of data which can be sent to a chat ([send_event](../../agent-api/client-server#send-event)). There are different types of events:
+
+ - [message](#message)
+ - [annotation](#annotation)
+ - [filled-form](#filled-form)
+ - [system message](#system-message)
+ - [file](#file)
+ - [custom event](#custom-event)
+
+
+[Incoming event](../../agent-api/server-client#incoming-event) push will inform you about events sent to a chat (on both, agent and customer side).
+
+
+### Common event fields
+
+All events have some common fields, we will describe them here to avoid duplication:
```

Comment:
All events have some common fields, we will describe them here to avoid duplication: -> może: Here are the fields shared by all events: